Simplenote/Classes/SPNoteListViewController+Extensions.swift

```diff
@@ -646,6 +646,13 @@ extension SPNoteListViewController {
 
     @objc(keyboardWillChangeFrame:)
     func keyboardWillChangeFrame(note: Notification) {
+        
+        guard self.view.window != nil else {
```

Comment:
Nipticky: no need for `self` in Swift (unless you're inside a Closure)
Extra nipticky: Swiftier way below!

```
guard let _ = view.window else {
   return
}
```